extension/src/fileSystem/index.ts

```diff
@@ -276,8 +277,9 @@ export const addPlotToDvcYamlFile = (cwd: string, plotObj: PlotConfigData) => {
   if (!plots?.range) {
     const plotYaml = getPlotsYaml(plotObj, dvcYamlLines)
     dvcYamlLines.push(...plotYaml)
-    writeFileSync(dvcYamlFile, dvcYamlLines.join('\n'))
-    return
```

Comment:
This "no plots found" route is actually missing the `openFileInEditor` line. I've added it and updated the tests to check for this function being called.